qemu/tests/cfg/live_backup.cfg

```diff
@@ -14,14 +14,14 @@
     image_name_incremental0 = "images/incremental0"
     clean_cmd = rm -f
     before_full_backup = "stop"
-    after_full_backup = "wait_for_finished resume"
+    after_full_backup = "resume"
     before_incremental = "stop"
-    after_incremental = "wait_for_finished verify_job_info"
+    after_incremental = "verify_job_info reopen"
     check_params = ""
     variants:
         - @default:
             before_full_backup = "create_files reboot stop"
-            after_incremental += " reopen verify_md5s"
+            after_incremental += " reopen verify_md5s verify_efficiency"
```

Comment:
@vivianQizhu  i prefer to move "reopen" out of the variants,  not only use it in "@default".  i think it is better to check it when use different granularity value , date_plane and other new option in the future. thanks !